### DISCUSSION TITLE
UID (slug) type edit issue

### DISCUSSION THREAD

**Discussion by waynehaffenden** [strapi/strapi#6763] [Legacy Topics]
I'm not sure whether this is an actual bug or feature. I'm using the UID/slug type for multiple collections to define page URLs for content. When editing that content type it generates a new slug incremented each time, the problem with this is that it means that pages then break as their URLs change. Obviously, if I had a page for a product for example /products/producta it would become /products/producta-1 etc which is not ideal especially for SEO.

When trying to override the slug during edit (removing -1) it fails saying the slug is already in use. Now this might be the correct way this field type should be used and therefore wondered if there was a config option to stop this behaviour and keep the slug to what was originally entered (unless explicitly changed)?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by waynehaffenden** [comment]
@derrickmehaffy Thank you, I wasn't sure the best place for this. Hoping to get an answer as previous discussions of mine have sadly had no joy.

**Reply by derrickmehaffy** [reply]
I don't believe there is a workaround for duplicating slugs to be honest, though I don't have a good answer for you on this one.
